### DISCUSSION TITLE
Parallel run in single node

### DISCUSSION THREAD

**Discussion by jack2gs** [elsa-workflows/elsa-core#3374] [Q&A] [answered]
Hi team,

Is one single node able to run workflows in parallel? I've tested it and found one single node will just run workflow one by one rather than in parallel. Could anyone confirm it please?

Thanks,
Jack

**Comment by sfmskywalker** [comment, marked as answer]
Yes, one node is able to run multiple workflows in parallel. For example, let's say you have a workflow definition with an HTTP trigger. If you fire 10 requests to this workflow, 10 instances will execute independent from one another (i.e. not consecutively, but in parallel - just like invoking a regular HTTP endpoint multiple times in parallel.

Another example, let's say you dispatch 20 workflows and have Rebus configured with `NumberOfWorkers = 4` and `MaxParallelism = 4`, then on a node with 4 CPUs it should be able to run 16 workflows in parallel (at least that's how I understood [Rebus parallelism](https://github.com/rebus-org/Rebus/wiki/Workers-and-parallelism#defaults). Even if I'm not entirely on point though, you can definitely run multiple workflows in parallel).
